test/tests/functional/pbs_qsub_opts_args.py

```diff
@@ -177,3 +193,55 @@ def test_qsub_with_option_a(self):
         except PbsSubmitError as e:
             self.validate_error(e)
         self.server.expect(JOB, {'job_state': 'R'}, id=jid_2)
+
+    @requirements(num_moms=2)
+    def test_qsub_sandbox_private_jobdir_shared(self):
+        """
+        Test submission of job with sandbox=PRIVATE,
+        and moms have $jobdir_root set to shared.
+        """
+        self.remove_jobdir = True
+        momA = self.moms.values()[0]
+        momB = self.moms.values()[1]
+
+        c = {'$jobdir_root': '%s shared' % self.jobdir_root}
+        l = {'$logevent': 4095}
+        for mom in [momA, momB]:
+            mom.add_config(c)
+            if mom == momB:
+                mom.add_config(l)
+            mom.restart()
+
+        a = {'Resource_List.select': '2:ncpus=1',
+             'Resource_List.place': 'scatter',
+             ATTR_sandbox: 'PRIVATE',
+             }
+        j = Job(TEST_USER, attrs=a)
+        j.set_sleep_time(10)
+        jid = self.server.submit(j)
+        self.server.expect(JOB, {'job_state': 'R'}, id=jid)
+        attribs = self.server.status(JOB, id=jid)
+        jobdir = attribs[0]['jobdir']
+        self.jobdir.append(jobdir)
+        relpath = os.path.join(
+            self.server.pbs_conf['PBS_EXEC'], 'bin', 'pbs_release_nodes')
```

Comment:
Isn't there a PTL wrapper for release nodes?

Reply:
None right now.